engine/flink/tests/src/test/scala/pl/touk/nussknacker/extensions/SpelExtension.scala

```diff
@@ -0,0 +1,11 @@
+package pl.touk.nussknacker.extensions
+
+import pl.touk.nussknacker.engine.graph.expression.Expression
+
+trait SpelExtension {
+
+  implicit class SpelExpresion(expression: String) {
```

Comment:
1. We already have `pl.touk.nussknacker.engine.spel.Implicits`. Does your intension is to icremenatally replace usage of it with a little bit more explicit converters syntax? If so, let's add a TODO in this class. 
2. It doesn't do any side effect so it can be parameterless.